### PR TITLE
Replacing unmaintained hard-coded Less library with a better option

### PR DESCRIPTION
**Pull request by escopecz**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

Replacing https://github.com/wikimedia/less.php with https://github.com/wikimedia/less.php

Installing the new library using Composer instead of hard-coding it to the Mautic's codebase. The old one had warnings in PHP 8.1 and so this is needed for PHP 8.1 support (https://github.com/mautic/mautic/pull/12382)

The deprecation this is solving: 
```
PHP Deprecated:  preg_match(): Passing null to parameter #4 ($flags) of type int is deprecated in /home/runner/work/mautic/mautic/plugins/MauticFocusBundle/Include/lessc.inc.php on line 3996
```

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test that the Focus items can modify the view of the modal or bar.

### CODE REVIEW

**Review by mollux — COMMENTED** [review]
Nice find!

Change makes sense, still needs a fix for the obsolete line in [`phpstan.neon`](https://github.com/mautic/mautic/blob/5.x/phpstan.neon#L32) file

**Review by mollux — APPROVED** [review]
changes look good and make sense!

File: composer.lock [hunk 1]
```diff
@@ -5824,6 +5825,7 @@
                         "[project-root]/Gruntfile.js": "assets/scaffold/files/Gruntfile.js",
                         "[project-root]/package.json": "assets/scaffold/files/package.json",
                         "[project-root]/package-lock.json": "assets/scaffold/files/package-lock.json",
+                        "[project-root]/modernizr-config.json": "assets/scaffold/files/modernizr-config.json",
```
Comment (mollux) [inline composer.lock:5828]: @escopecz I will add an extra github action check to ensure the `composer.json` and `composer.lock` 
 file are in sync. This change is unrelated to this PR, but can be confusing when doing QA

Comment (escopecz) [reply composer.lock:5828]: Thanks!

**Pull request merged**